wireguard/map.jinja

```diff
@@ -13,9 +13,9 @@
     merge=salt['grains.filter_by'](osfamilymap, grain='os_family',
       merge=salt['grains.filter_by'](osmap, grain='os',
         merge=salt['grains.filter_by'](osfingermap, grain='osfinger',
-          merge=salt['grains.filter_by'](salt['grains.filter_by'](osarchmap, grain='osfinger'), grain='osarch')
+          merge=salt['grains.filter_by'](salt['grains.filter_by'](osarchmap, grain='osfinger'), grain='osarch',
+            merge=salt['pillar.get']('wireguard'))
```

Comment:
I'd rather use the "well known" `lookup` key as in `wireguard:lookup`. What do you think?

Reply:
I'm unfamiliar; but presumably you're talking about this?

https://docs.saltstack.com/en/latest/topics/development/conventions/formulas.html#abstracting-static-defaults-into-a-lookup-table

If so, I'll review that and update the PR.

Reply:
Yes exactly.

Reply:
I made the tiny update, and it appears to be working as expected in my environment. Let me know if you want it squashed before merge, or anything else.